post-scan-actions/aws-python-teams-notification/handler.py

```diff
@@ -0,0 +1,81 @@
+import urllib3
+import json
+import os
+http = urllib3.PoolManager()
+import textwrap
+
+def lambda_handler(event, context):
+
+    url = os.environ['TEAMS_URL']
+    channel = os.environ['TEAMS_CHANNEL']
```

Comment:
It's not used anymore. Please remove the environment variable as well.

Reply:
removed 'channel' env variable